packages/core/package.json

```diff
@@ -29,8 +30,7 @@
     "build:typedocs:asjson": "typedoc --options typedoc.asjson.json",
     "lint": "eslint 'src/**/*.ts'",
     "lint:fix": "yarn run lint --fix",
-    "run:nodes": "docker-compose -f docker-compose.yml up",
-    "clean": "rm -rf ./dist ./dist-cjs ./dist-esm ./types"
+    "run:nodes": "docker-compose -f docker-compose.yml up"
```

Comment:
which one is the target docker file?

Reply:
Here I just moved "clean" method call to the top. But indeed we don't have a docker in our repo, so we can remove the inherited code.